collector/nvidia.go

```diff
@@ -0,0 +1,305 @@
+// +build windows
```

Comment:
I think we might want to use some extra tag (eg nvidia) here, since most people probably won't have the necessary libraries to build this locally.

Reply:
This module does not link against any static libs (since the NVML package from NVIDIA simply does not have them) and loads the needed dll via C.LoadLibrary so if it is not there it is just not there. No actual libraries are needed for building.

Reply:
Ah, ok then :+1: